definitions/pipelines/rnaseq_star_fusion.cwl

```diff
@@ -172,3 +179,10 @@ steps:
             bam: mark_dup/sorted_bam
         out:
             [metrics, chart]
+    generate_multiqc:
+        run: ../tools/multiqc.cwl
+        in:
+            inputfiles_array:[generate_qc_metrics/metrics,star_align_fusion/log_final]
```

Comment:
```suggestion
            inputfiles_array: [generate_qc_metrics/metrics,star_align_fusion/log_final]
```

Reply:
@sridhar0605 - should probably either accept or reject this suggestion. (i don't actually know if the space matters...)